cronjobs.yml

```diff
@@ -15,7 +15,15 @@ cronJobs:
         memory: "5Mi"
       limits: # maximum allowed resources
         cpu: "1.5"
-        memory: "600Mi"
+        memory: "1.5Gi"
+    # kanopy recommends all cronjobs run on batch instance groups
```

Comment:
@zamj pointed out that the Kanopy team now recommends that all cronjobs run on batch instance groups (and I confirmed with someone on Kanopy that they are updating their [docs](https://github.com/10gen/kanopy-docs/blob/master/docs/batch.md) to reflect this stance)